src/units/formula_manager.cpp

```diff
@@ -28,16 +29,16 @@ void unit_formula_manager::add_formula_var(std::string str,wfl:: variant var)
 
 void unit_formula_manager::read(const config & ai)
 {
-	unit_formula_ = ai["formula"].str();
-	unit_loop_formula_ = ai["loop_formula"].str();
-	unit_priority_formula_ = ai["priority"].str();
+	unit_formula_ = ai.get_deprecated_attribute("formula", "unit][ai", DEP_LEVEL::FOR_REMOVAL, "FormulaAI will be removed in 1.17").str();
+	unit_loop_formula_ = ai.get_deprecated_attribute("loop_formula", "unit][ai", DEP_LEVEL::FOR_REMOVAL, "FormulaAI will be removed in 1.17").str();
+	unit_priority_formula_ = ai.get_deprecated_attribute("priority", "unit][ai", DEP_LEVEL::FOR_REMOVAL, "FormulaAI will be removed in 1.17").str();
```

Comment:
Should there be more square brackets in all of these `unit][ai` strings?

Reply:
I did it this way because it's later surrounded by brackets in the actual message.